src/Microsoft.PowerShell.Commands.Management/commands/management/GetComputerInfoCommand.cs

```diff
@@ -24,7 +26,6 @@ namespace Microsoft.PowerShell.Commands
     /// </summary>
     [Cmdlet(VerbsCommon.Get, "ComputerInfo",
         HelpUri = "https://go.microsoft.com/fwlink/?LinkId=799466")]
-    [Alias("gin")]
```

Comment:
My opinion doesn't stem from the performance concern. I just think we should no longer keep stuffing unnecessary settings to `InitialSessionState.cs`. Keep alias declaration in individual cmdlet is good for at least 2 reasons:
1. alias declaration is together with the cmdlet definition -- easy to read/discover and the alias exists in the cmdlet as a metadata information which can be analyzed using reflection by tools (if there is any tool doing so).
2. we want to stop the pattern of declaring aliases in a centralized location, so that when multiple people working on different cmdlets at the same time, they will have less conflicts that are resulted from editing the same file.

For those existing alias declarations in `InitialSessionState.cs`, we cannot move them out due to the `ScopeItemOption` settings (BTW, I think for some of the aliases there, the authors probably just copied the existing options without knowing if they should actually be applied). But for new aliases, I think it's better to have them coupled with the cmdlet unless an option other than 'none' is really desired.